### DISCUSSION TITLE
How to extract IPTC metadata from PNG files?

### DISCUSSION THREAD

**Discussion by dhoffer** [drewnoakes/metadata-extractor#672] [General]
This is what is returned.

Raw profile type iptc:
IPTC profile
      36
3842494d04040000000000181c0219000c74657374312c2074657374321c020000020004

So how can I decode this to get the actual metadata tags?  My understanding (limited) is that these are stored in Photoshop metadata format inside the PNG.

The actual metadata of the above as reported by exiftool.

Keywords: test1, test2

How can I decode the same with metadata-extractor?

**Comment by drewnoakes** [comment]
Hard to say without a sample image to look at. Can you provide one?

**Reply by dhoffer** [reply]
![Standard_withKeywords](https://github.com/user-attachments/assets/ffeefdc4-1f4c-4311-91aa-c0afb5fe8773)

Sure.  I just attached a sample PNG with the keywords test1, test2 (via exiftool).

**Reply by drewnoakes** [reply]
Thanks.

I've converted this into an issue so that we can track the work: https://github.com/drewnoakes/metadata-extractor/issues/673

If you'd like to work on this I'm happy to provide some pointers.
